package.json

```diff
@@ -12,28 +12,37 @@
   "@wordpress/dom-ready": "2.0.2",
   "@wordpress/i18n": "3.1.1",
   "babel-core": "6.26.3",
+  "babel-eslint": "10.0.1",
   "babel-loader": "7.1.5",
   "babel-plugin-transform-object-rest-spread": "6.26.0",
   "babel-plugin-transform-react-jsx": "6.24.1",
   "babel-plugin-transform-runtime": "6.23.0",
   "babel-preset-env": "1.7.0",
   "cross-env": "5.2.0",
-  "eslint": "4.19.1",
+  "eslint": "5.14.0",
   "eslint-config-wordpress": "2.0.0",
-  "eslint-plugin-react": "7.11.1",
+  "eslint-plugin-react": "7.12.4",
   "grunt": "1.0.2",
   "grunt-contrib-clean": "1.1.0",
   "grunt-contrib-copy": "1.0.0",
   "grunt-shell": "2.1.0",
   "grunt-wp-deploy": "1.2.1",
+  "npm-run-all": "^4.1.5",
   "timeago.js": "3.0.2",
   "webpack": "4.29.5",
   "webpack-cli": "3.2.3"
  },
  "main": "blocks/index.js",
  "scripts": {
-  "build": "grunt build && grunt create-build-zip",
+  "build": "npm-run-all build:*",
```

Comment:
Is this going to be stable? Could it not result in `zip` being run before `prepare`? Objects in JS do not have ordered keys, unless NPM explicitly sorts them by key when globbing like this. Do you have confirmation?

Reply:
So far I have never had any problems with the order being mixed up by `npm-run-all` or something.

> Objects in JS do not have ordered keys

I thought this changed a bit with ES2015?

> Do you have confirmation?

According to the author, it executes tasks in definition order in `package.json`. https://github.com/mysticatea/npm-run-all/issues/116#issuecomment-348672125

I haven't looked at the code of NPM or the package though to see how order is determined.